### PR TITLE
Fix default option behaviour

### PR DESCRIPTION
**Pull request by GiantRobots**
## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,11 +42,11 @@
     "preview:local": "VITE_TEMPORAL_UI_BUILD_TARGET=local vite preview --port 3000",
     "preview:docker": "VITE_API=http://localhost:8080 VITE_TEMPORAL_UI_BUILD_TARGET=local vite preview --port 3000",
     "preview:cloud": "VITE_TEMPORAL_UI_BUILD_TARGET=cloud vite preview --port 3000",
-    "package": "svelte-package && rsync -r ./package/lib/ ./package/ && rm -rf ./package/lib",
+    "package": "svelte-package",
     "package:patch": "pnpm version patch && svelte-package",
     "package:minor": "pnpm version minor && svelte-package",
     "package:major": "pnpm version major && svelte-package",
-    "story:dev": "histoire dev",
+    "story:dev": "histoire dev --port 3030",
```
Comment (rossedfort) [inline package.json:49]: why the port change?

File: src/lib/holocene/primitives/menu/menu.svelte [hunk 2]
```diff
@@ -7,20 +7,25 @@
...
+>
+  <slot />
+</ul>
 
 <style lang="postcss">
+  /* We should probably have a nonVisible class here that allows screen readers to see the content.
+  Which might be an option on the menu (for things like split buttons) */
+  .hide {
+    display: none;
+  }
```
Comment (rossedfort) [inline src/lib/holocene/primitives/menu/menu.svelte:27]: maybe `sr-only` ? https://tailwindcss.com/docs/screen-readers

**Review by rossedfort — APPROVED** [review]
Looks good

File: src/lib/holocene/select/select.story.svelte [hunk 3]
```diff
@@ -79,6 +79,24 @@
...
+      <OptionGroup label="Reptiles">
+        <Option value={4}>Snake</Option>
+        <Option value={5}>Lizard</Option>
+      </OptionGroup>
+      <OptionGroup label="Amphibians">
+        <Option value={6}>Frog</Option>
+        <Option value={7}>Salamander</Option>
+      </OptionGroup>
+    </Select>
+  </Hst.Variant>
```
Comment (rossedfort) [inline src/lib/holocene/select/select.story.svelte:98]: Probably can 🚮

File: package.json (hunk 1 above)
Comment (GiantRobots) [reply package.json:49]: I was on an airplane and the console outputted so much stuff I couldn't find the port it was running on and I didn't have internet so I just set it I'll revert.

File: src/lib/holocene/select/select.story.svelte (hunk 3 above)
Comment (GiantRobots) [reply src/lib/holocene/select/select.story.svelte:98]: This is essentially a test case to verify that the changes we implemented work. I think I'd prefer to keep it around.

Comment (rossedfort) [reply src/lib/holocene/select/select.story.svelte:98]: Oh I didn't notice these `<Option />`'s had `value`'s, sounds good 👍

**Pull request merged**